web-marketplace/src/components/organisms/Order/Order.Summary.tsx

```diff
@@ -0,0 +1,189 @@
+              />
+            </div>
+          }
+          className="items-baseline"
+        />
+      </OrderSummarySection>
+      <OrderSummarySection
+        icon={<CertifiedDocumentIcon className="text-grey-500" />}
+        title={_(msg`Retirement Info`)}
+      >
+        {isTradable ? (
+          <OrderSummaryRow
+            title={_(msg`Tradable Credits`)}
+            value={<span className="italic">{tradableCredits}</span>}
+            clampDescription={false}
+          />
+        ) : (
+          <>
+            <OrderSummaryRow title={_(msg`Retirement reason`)} value={reason} />
+            <OrderSummaryRow
+              title={_(msg`Retirement location`)}
+              value={location}
+            />
+            {/* <OrderSummaryRow
+              title="Make anonymous"
+              value={
+                <MakeAnonymous
+                  value={data.makeAnonymous ? 'Yes' : 'No'}
+                  onChange={() => []}
+                  ariaLabel="anonymous-select"
+                  name="anonymous-select"
+                />
+              }
+            /> */}
+          </>
+        )}
+      </OrderSummarySection>
+      <OrderSummarySection
+        icon={<PaymentInfoIcon className="text-grey-500" />}
+        title={_(msg`payment info`)}
+      >
+        {isCardPayment ? (
+          <>
+            <OrderSummaryRow title={_(msg`name on card`)} value={nameOnCard} />
+            <OrderSummaryRow
+              title={_(msg`card info`)}
+              value={
+                <span>
+                  {_(msg`Visa ending in`)} {cardLast4}
+                </span>
+              }
+              className="items-center"
+            />
+          </>
+        ) : (
+          <OrderSummaryRow
+            title={_(msg`payment currency`)}
+            value={<DenomIconWithCurrency currency={askDenom} />}
+          />
+        )}
+      </OrderSummarySection>
+      <OrderSummarySection
+        icon={<BlockchainIcon className="text-grey-500" />}
+        title={_(msg`Blockchain Details`)}
+      >
+        <OrderSummaryRow title={_(msg`purchase date`)} value={purchaseDate} />
+        {blockchainRecord && (
+          <OrderSummaryRow
+            title={
+              <div className="relative">
+                <span className="inline">
+                  <Trans>blockchain record</Trans>
+                </span>
+                <QuestionMarkTooltip
+                  className="absolute bottom-0 right-20"
+                  title={_(
+                    msg`A unique identifier that tracks and verifies a specific transaction on the blockchain.`,
+                  )}
+                />
+              </div>
+            }
+            value={
+              <Link
+                href="https://google.com"
+                className="text-grey-700 !underline inline-block text-ellipsis w-[100px] overflow-hidden"
+              >
+                {blockchainRecord}
+              </Link>
```

Comment:
we should use the same logic as in https://github.com/regen-network/regen-web/blob/d016f292e058c61b73f9947c80f7fb8184c79a8c/web-marketplace/src/components/organisms/ConfirmationModal.tsx#L134-L144